petric.py

```diff
@@ -37,9 +37,11 @@
 log = logging.getLogger('petric')
 TEAM = os.getenv("GITHUB_REPOSITORY", "SyneRBI/PETRIC-").split("/PETRIC-", 1)[-1]
 VERSION = os.getenv("GITHUB_REF_NAME", "")
-OUTDIR = Path(f"/logs/{TEAM}/{VERSION}" if TEAM and VERSION else "./output")
-if not (SRCDIR := Path("/mnt/share/petric")).is_dir():
-    SRCDIR = Path("./data")
+_repo_directory = Path(__file__).parent
+OUTDIR = Path(os.getenv("PETRIC_OUTDIR", f"/logs/{TEAM}/{VERSION}" if TEAM and VERSION else _repo_directory / "output"))
+SRCDIR = Path(os.getenv("PETRIC_SRCDIR", "/mnt/share-public/petric/2"))
+if not SRCDIR.is_dir():
+    SRCDIR = _repo_directory / 'data'
```

Comment:
```suggestion
_petric_path = Path(__file__).parent
OUTDIR = Path(os.getenv("PETRIC_OUTDIR", f"/logs/{TEAM}/{VERSION}" if TEAM and VERSION else _petric_path / "output"))
SRCDIR = Path(os.getenv("PETRIC_SRCDIR", "/mnt/share/petric"))
if not SRCDIR.is_dir():
    SRCDIR = _petric_path / 'data'
```

More future-proof to symlink `ln -s /mnt/share-public/petric/2 /mnt/share/petric`?